.circleci/testcafe.json

```diff
@@ -1,5 +1,5 @@
 {
-  "src": "tests/acceptance/acceptancesuites/*.js",
+  "src": ["tests/acceptance/acceptancesuites/*.js", "!tests/acceptance/acceptancesuites/searchbaronlysuite.js"],
```

Comment:
I know this code has been approved already, but would it make sense to put the searchbaronlysuite into its own folder? Really nitpicking here but I could see a new person inheriting the SDK, and being confused why only some of the acceptance tests would run by default